src/include/duckdb/common/map.hpp

```diff
@@ -9,8 +9,10 @@
 #pragma once
 
 #include <map>
+#include <unordered_map>
 
 namespace duckdb {
 using std::map;
 using std::multimap;
+using std::unordered_map;
```

Comment:
We already have `src/include/duckdb/common/unordered_map.hpp`, these should be imported separately.